### PR TITLE
Page surgery and remove to fixes to known issues

### PR DESCRIPTION
**Pull request by ElleEss**
#### Description
* UTY-1778
* Removed protocol logging known issue (still being validated)
* Removed IL2cPP known issue
* Formatting surgery

#### Tests
Improbadoc

#### Documentation
Is docs.

**Did you remember a changelog entry?**
I wish we had this for docs.

#### Primary reviewers
RobS, Jess/Jamie, Martijn.

### CODE REVIEW

File: docs/known-issues.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
 # SpatialOS GDK for Unity known issues
 
-Known issue = any major user-facing bug or lack of user-facing feature that:
-1. diverges from vanilla Unity ECS design or implementation **OR**
-1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+A known issue is any major user-facing bug or lack of user-facing feature that:
+
+* Diverges from vanilla Unity ECS design or implementation. 
+<br/>**OR**
+* Diverges from user expectations of a SpatialOS project (for example; interacting across worker instance boundaries).
```
Comment (jamiebrynes7) [inline docs/known-issues.md]: I still think this definition isn't correct 🤷‍♂️

File: docs/known-issues.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-| Protocol logging on Linux workers is disabled due to a libcpp linking bug in Unity.                                                                                                   | 2019/01/28 | None                                                                                                | No workaround.                                                |
+| The Unity Editor doesn't correctly import NavMeshes.                                                                                                                                                                                                                                                                         | 2018/10/12 | None                                                                                                   | Manually regenerate the NavMesh after opening your project.            |
```
Comment (jamiebrynes7) [inline docs/known-issues.md]: I think the nav mesh one is gone as well!

**Comment by jamiebrynes7** [comment]
Please please please do not merge this until https://github.com/spatialos/gdk-for-unity/pull/757 is confirmed and merged

File: docs/known-issues.md (hunk 1 above)
Comment (NotRobSheridan) [reply docs/known-issues.md]: I feel like 'known issue' speaks for itself... maybe we could look at changing this to something like: 

>This page outlines known issues that may be fixed in later releases of the GDK. Use this page to see if we are already aware of an issue that may be affecting you. If your issue is not listed here, you can report a bug [here](bug report link)

Or something like that. At the moment it looks like we're trying to redefine the widely known definition of 'known issue'.

**Comment by zeroZshadow** [comment]
Protocol logging is validated, works again

File: docs/known-issues.md (hunk 2 above)
Comment (zeroZshadow) [reply docs/known-issues.md]: Correct

File: docs/known-issues.md (hunk 1 above)
Comment (jessicafalk) [reply docs/known-issues.md]: I really like that version! definitely feels more appropriate than trying to define what a known issue is

Comment (ElleEss) [reply docs/known-issues.md]: No idea who wrote this or when. Things just appear in the docs by magic. <"Expecto words!">
I don't know if it was in response to someone's confusion or a request from on high? Checking the blame and seeing who and why.

File: docs/known-issues.md [hunk 3]
```diff
@@ -1,14 +1,14 @@
 # SpatialOS GDK for Unity known issues
 
-Known issue = any major user-facing bug or lack of user-facing feature that:
-1. diverges from vanilla Unity ECS design or implementation **OR**
-1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+A known issue is any major user-facing bug or lack of user-facing feature that:
```
Comment (ElleEss) [inline docs/known-issues.md]:
```suggestion

```

File: docs/known-issues.md (hunk 2 above)
Comment (jamiebrynes7) [reply docs/known-issues.md]: @ElleEss ^^

File: docs/known-issues.md [hunk 4]
```diff
@@ -1,14 +1,13 @@
 # SpatialOS GDK for Unity known issues
 
-Known issue = any major user-facing bug or lack of user-facing feature that:
-1. diverges from vanilla Unity ECS design or implementation **OR**
-1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+
+* Diverges from vanilla Unity ECS design or implementation. 
+<br/>**OR**
+* Diverges from user expectations of a SpatialOS project (for example; interacting across worker instance boundaries).
```
Comment (ElleEss) [inline docs/known-issues.md]:
```suggestion
Something in the GDK not working as expected? These are the known issues; later releases of the GDK may contain fixes for these.  
If your issue is not listed here, you can raise an issue on through our [GitHub repo](https://github.com/spatialos/unrealgdk/issues).
```

File: docs/known-issues.md [hunk 5]
```diff
@@ -1,14 +1,14 @@
 # SpatialOS GDK for Unity known issues
 
-Known issue = any major user-facing bug or lack of user-facing feature that:
-1. diverges from vanilla Unity ECS design or implementation **OR**
-1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+
+* Diverges from vanilla Unity ECS design or implementation.
```
Comment (ElleEss) [inline docs/known-issues.md]:
```suggestion
```

File: docs/known-issues.md [hunk 6]
```diff
@@ -1,14 +1,13 @@
 # SpatialOS GDK for Unity known issues
 
-Known issue = any major user-facing bug or lack of user-facing feature that:
-1. diverges from vanilla Unity ECS design or implementation **OR**
-1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+
+<br/>**OR**
```
Comment (ElleEss) [inline docs/known-issues.md]:
```suggestion
```

File: docs/known-issues.md [hunk 7]
```diff
@@ -1,14 +1,13 @@
...
-| Protocol logging on Linux workers is disabled due to a libcpp linking bug in Unity.                                                                                                   | 2019/01/28 | None                                                                                                | No workaround.                                                |
+| The Unity Editor doesn't correctly import NavMeshes.                                                                                                                                                                                                                                                                         | 2018/10/12 | None                                                                                                   | Manually regenerate the NavMesh after opening your project.            |
```
Comment (ElleEss) [inline docs/known-issues.md]:
```suggestion
```

File: docs/known-issues.md [hunk 8]
```diff
@@ -1,14 +1,11 @@
 # SpatialOS GDK for Unity known issues
 
-Known issue = any major user-facing bug or lack of user-facing feature that:
-1. diverges from vanilla Unity ECS design or implementation **OR**
-1. diverges from user expectations from a SpatialOS project (e.g. interacting across worker boundaries)
+
+Something in the GDK not working as expected? These are the known issues; later releases of the GDK may contain fixes for these.  
+If your issue is not listed here, you can raise an issue on through our [GitHub repo](https://github.com/spatialos/unrealgdk/issues).
```
Comment (jessicafalk) [inline docs/known-issues.md]:
```suggestion
If your issue is not listed here, you can raise an issue through our [GitHub repo](https://github.com/spatialos/unrealgdk/issues).
```

File: docs/known-issues.md [hunk 9]
```diff
@@ -1,14 +1,11 @@
...
```
Comment (ElleEss) [inline docs/known-issues.md]: @oblm Is this OK?

File: docs/known-issues.md [hunk 10]
```diff
@@ -1,14 +1,11 @@
...
-| Protocol logging on Linux workers is disabled due to a libcpp linking bug in Unity.                                                                                                   | 2019/01/28 | None                                                                                                | No workaround.                                                |
+| Running the FPS Starter Project in the Unity Editor with the current platform as Android leads to the walls being slightly transparent.                                                                                                      | 2019/01/28 | None                                                                                                | It renders correctly when running in the Emulator or an Android device                                                |
```
Comment (jessicafalk) [inline docs/known-issues.md]:
```suggestion
| Running the FPS Starter Project in the Unity Editor with the current platform as Android leads to the walls being slightly transparent.                                                                                                      | 2019/01/28 | None                                                                                                | It renders correctly when running in the Emulator or an Android device.                                                |
```

File: docs/known-issues.md [hunk 11]
```diff
@@ -1,14 +1,11 @@
...
+| Building a worker in the FPS Starter Project for Linux will throw errors when compiling the `Hidden/PostProcessing/FinalPass` shader.                                                                                                    | 2019/01/28 | None                                                                                                | The build still succeeds and won't affect your worker.                                                |
```
Comment (jessicafalk) [inline docs/known-issues.md]:
```suggestion
| Building a worker in the FPS Starter Project for Linux throw errors when compiling the `Hidden/PostProcessing/FinalPass` shader.                                                                                                    | 2019/01/28 | None                                                                                                | The build still succeeds and won't affect your worker.                                                |
```

Comment (ElleEss) [reply docs/known-issues.md]: If we get rid of "will" then we need to make "throw" into "throws" so accepting the change and then re-editing. And I don't like "will" either.

File: docs/known-issues.md (hunk 9 above)
Comment (ElleEss) [reply docs/known-issues.md]: Has said yes on Slack.

File: docs/known-issues.md [hunk 12]
```diff
@@ -1,14 +1,11 @@
...
+| Running the FPS Starter Project in the Unity Editor with the current platform as Android leads to the walls being slightly transparent.                                                                                                      | 2019/01/28 | None                                                                                                | It renders correctly when running in the Emulator or an Android device.                                                |
+| Building a worker in the FPS Starter Project for Linux throw errors when compiling the `Hidden/PostProcessing/FinalPass` shader.                                                                                                    | 2019/01/28 | None                                                                                                | The build still succeeds and won't affect your worker.                                                |
```
Comment (ElleEss) [inline docs/known-issues.md]:
```suggestion
| Building a worker in the FPS Starter Project for Linux throws errors when compiling the `Hidden/PostProcessing/FinalPass` shader.                                                                                                    | 2019/01/28 | None                                                                                                | The build still succeeds and won't affect your worker.                                                |
```

File: docs/known-issues.md [hunk 13]
```diff
@@ -1,14 +1,11 @@
...
-| Protocol logging on Linux workers is disabled due to a libcpp linking bug in Unity.                                                                                                   | 2019/01/28 | None                                                                                                | No workaround.                                                |
+| Running the FPS Starter Project in the Unity Editor with the current platform as Android leads to the walls being slightly transparent.                                                                                                      | 2019/01/28 | None                                                                                                | It renders correctly when running in the Emulator or an Android device.                                                |
```
Comment (ElleEss) [inline docs/known-issues.md]:
```suggestion
| Running the FPS Starter Project in the Unity Editor with the **current platform** set as Android leads to the walls being slightly transparent.                                                                                                      | 2019/01/28 | None                                                                                                | It renders correctly when running in the Emulator or an Android device.                                                |
```

**Pull request merged**